fossology/fossology
# feat(project): add readme_oss reprot generation code in project view

## Description

add readme_oss reprot generation code in project view

### Changes

I have made some modification of readme_oss generate plugin. For now, it transform a new paramater "type" to schedule. 

## How to test

Click the ReadMe_OSS generation button on BrowseFile page or BrowseProject page. Then open the Dbeaver to look at the jobqueue table, you will see the new paramater in jq_cmd_args column.


<a href="https://gitpod.io/#https://github.com/fossology/fossology/pull/2314"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
